lib/Pod/Simple/XHTML.pm

```diff
@@ -131,11 +131,12 @@ default.
 
 In turning C<< L<crontab(5)> >> into http://whatever/man/1/crontab, what
 to put before the "1/crontab". The default value is
```

Comment:
That should probably say `… into http://whatever/man/1/crontab.1` and `… before the "1/crontab.1"` because we now add another section number after the page name.